modules/nw-anp-banp-metrics.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included:
+// networking/network_security/AdminNetworkPolicy/nw-ovn-k-anp-banp-metric.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="anp-banp-metrics_{context}"]
+= Metrics
+
+[cols="1,1,1"]
+|===
+| Name | Labels |Explanation
```

Comment:
We usually see "Description" as the heading name for columns like this, just wondering if that would be appropriate here too?